tests/unit/amo/components/TestAddonReview.js

```diff
@@ -444,20 +443,5 @@ describe(__filename, () => {
           .then(() => mockUtils.verify());
       });
     });
-
-    describe('setDenormalizedReview', () => {
-      it('dispatches a setReview action', () => {
-        const review = {
-          ...defaultReview,
-          body: 'some body',
-        };
-        actions.setDenormalizedReview(review);
```

Comment:
We still need this test because it covers how the component [exposes a function](https://github.com/mozilla/addons-frontend/blob/5cdc08bf0cccb083f487aed8de70e449b37cd0b6/src/amo/components/AddonReview/index.js#L283) for dispatching the action.

It's odd way to write a component. I believe it was originally done this way for ease of testing in a world that predated `shallowUntilTarget` (where `sinon.stub(store, 'dispatch')` was not an option).